odl/test/tomo/geometry/geometry_test.py

```diff
@@ -34,7 +34,7 @@
 det_pos_init_3d = simple_fixture('det_pos_init', det_pos_init_3d_params)
 axis = simple_fixture('axis', det_pos_init_3d_params)
 shift = simple_fixture('shift', [0, 1])
-curved_det = simple_fixture('curved_det', [False, True])
+detector_type = simple_fixture('detector_type', [0, 1, 2])
```

Comment:
Minor quibble: it would be easier for a later maintainer of the tests to have explicit names for the `detector_type`, rather than numbers. Even "flat", "curved1", "curved2" would be better than 0,1,2.